build.wake

```diff
@@ -154,11 +154,40 @@ def tarball Unit =
     # Identify those sources files to include in the tarball
     require Pass allSources = sources "." `.*`
 
-    def srcs =
+    def skipFolders =
+        (".circleci/", "debian/", ".github/", ".vscode/", Nil)
+        | catWith "|"
+
+    def skipFiles =
+        (
+            ".clang-format",
+            ".dockerignore",
+            ".git-blame-ignore-revs",
+            ".gitattributes",
+            "wake.spec.in",
+            ".wakemanifest",
+            Nil
+        )
+        | catWith "|"
+
+    def skipFilesRecursive =
+        (".gitignore", Nil)
+        | catWith "|"
+
+    require Pass regex =
+        "(({skipFolders}).*|({skipFiles})|(.*/)*({skipFilesRecursive}))"
+        | stringToRegExp
+
+    def Pair skips srcs =
         allSources
-        | filter (! matches `(debian/|wake.spec).*` _.getPathName) # omit packaging files
-        | filter (! matches `(\.circleci/|\.github/).*|\.dockerignore` _.getPathName) # omit CI files
-        | filter (! matches `(.*/)*\.gitignore` _.getPathName) # omit git files
+        | splitBy (matches regex _.getPathName)
```

Comment:
And then down here it would be `splitBy (\x exists (matches _ x.getPathName) toExclude)`